### PR TITLE
Even more logging improvements. 

### PR DESCRIPTION
**Pull request by LukaszRozmej**
## Changes

- Better spacing. 
- Remove decimals on Blk/s. 
- Introduce block numbers in ForkChoice message.

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [x] Other: logs

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

#### Notes on testing

### CODE REVIEW

**Comment by LukaszRozmej** [comment]
image

File: src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs [hunk 1]
```diff
@@ -213,15 +220,15 @@ public Task<ResultWrapper<ForkchoiceUpdatedV1Result>> Handle(ForkchoiceStateV1 f
         if (shouldUpdateHead)
         {
             _poSSwitcher.ForkchoiceUpdated(newHeadBlock.Header, forkchoiceState.FinalizedBlockHash);
-            if (_logger.IsInfo) _logger.Info($"Synced chain Head to  {newHeadBlock.ToString(Block.Format.Short)}");
+            if (_logger.IsInfo) _logger.Info($"Synced chain Head to {newHeadBlock.ToString(Block.Format.FullHashAndNumber)}");
```
Comment (benaadams) [inline src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs:223]: Wondering if should go other way and use `.Short` for both received new block and Synced and only use FullHash for errors? Is it useful?

Invalid block might want to copy paste the whole Hash; for valid blocks is 64 random hex bytes just noise?

**Review by benaadams — APPROVED** [review]

File: src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs (hunk 1 above)
Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs:223]: Dunno generally I hate short hashes when investigating any issue. It is acceptable if you have block number, but even then suboptimal.

Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs:223]: But now they align well with Received :)
So either both or none?

Comment (LukaszRozmej) [reply src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs]: Ok lets do both

Comment (benaadams) [reply src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs]: > Dunno generally I hate short hashes when investigating any issue. It is acceptable if you have block number, but even then suboptimal.

I think any errors should still use long hash

Comment (benaadams) [reply src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs]: But repeating the long hash lots and lots when is success seems like noise?

**Pull request merged**